tests/integration-tests/tests/schedulers/test_slurm/test_slurm_mpi/mpi_job.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+#SBATCH --export=ALL
+#SBATCH --nodes=5
+#SBATCH --ntasks-per-node=3
+#SBATCH --job-name=imb5
+#SBATCH --time=1
+#SBATCH --output=runscript.out
+
+module load intelmpi
```

Comment:
what about using openmpi that is available on all OSes? 
On centos6 it is probably not available as a modulefile if I recall correctly. Take a look at the `_test_mpi` function in the `common/mpi_common.py` module